examples/tutorials/traits_4.0/interfaces/adaptation.py

```diff
@@ -235,7 +235,8 @@ class Apartment(HasTraits):
 
 # --<Imports>------------------------------------------------------------------
 
-from traits.api import *
+from traits.api import (
+    Adapter, adapts, HasTraits, Instance, Interface, Str, Supports)
```

Comment:
Orthogonal: On this branch, I get this Import error:
```
$ python examples/tutorials/traits_4.0/interfaces/
/Users/kchoi/Work/ETS/py39-venv/traits-env/bin/python: can't find '__main__' module in '/Users/kchoi/Work/ETS/traits/examples/tutorials/traits_4.0/interfaces/'
(traits-env) kchoi-mbp2019:traits kchoi$ python examples/tutorials/traits_4.0/interfaces/adaptation.py 
Traceback (most recent call last):
  File "/Users/kchoi/Work/ETS/traits/examples/tutorials/traits_4.0/interfaces/adaptation.py", line 238, in <module>
    from traits.api import (
ImportError: cannot import name 'adapts' from 'traits.api' (/Users/kchoi/Work/ETS/traits/traits/api.py)
```

On master, I get this:
```
$ python examples/tutorials/traits_4.0/interfaces/adaptation.py 
Traceback (most recent call last):
  File "/Users/kchoi/Work/ETS/traits/examples/tutorials/traits_4.0/interfaces/adaptation.py", line 256, in <module>
    class PersonINameAdapter(Adapter):
  File "/Users/kchoi/Work/ETS/traits/examples/tutorials/traits_4.0/interfaces/adaptation.py", line 259, in PersonINameAdapter
    adapts(Person, IName)
NameError: name 'adapts' is not defined
```

Reply:
Good catch, thanks. Indeed we need to either delete or rewrite this particular example file (and similarly for `interfaces.py`). I think that's covered by the existing issue #437.